common/build/Jenkinsfile.pauli

```diff
@@ -91,30 +91,30 @@ pipeline {
         }
         unstable {
             echo 'DEBUG: Build is unstable'
-            emailext to: 'galexv+jenkins.status@umich.edu',
-                     recipientProviders: [brokenTestsSuspects(), culprits(), requestor()],
-                     subject: 'ALPSCore: Jenkins build is unstable',
-                     attachLog: true,
-                     compressLog: true,
-                     body: "ALPSCore build is unstable: see attached log"
+            //emailext to: 'galexv+jenkins.status@umich.edu',
```

Comment:
So what happens now when a test fails? Nothing?

Reply:
I am not sure it actually worked in the first place. And the last time I checked jenkins, this actually caused an error and test failure because it couldn't send an email.

Also somehow github stopped reading status from our jenkins and I am trying to understand why.